sql/10dbChildCareWaiverForm.sql

```diff
@@ -0,0 +1,49 @@
+-- Table for the dbChildCareWaiverForm
+CREATE TABLE dbChildCareWaiverForm (
+    id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    child_id INT NOT NULL,
+    child_first_name VARCHAR(256),
+    child_last_name VARCHAR(256),
+    birth_date DATE,
+    gender VARCHAR(6),
+    child_address VARCHAR(256),
+    child_city VARCHAR(100),
+    child_state VARCHAR(100),
+    child_zip VARCHAR(10),
+    medical_issues TEXT,
+    foods_to_avoid TEXT,
+    
```

Comment:
medical_issues and food_to_avoid will be in their own tables to keep everything normalized, so you can delete these two fields